src/main/java/core/basesyntax/Colors.java

```diff
@@ -0,0 +1,14 @@
+package core.basesyntax;
+
+public enum Colors {
```

Comment:
Enum Names: Enum types should be named using singular nouns
```suggestion
public enum Color {
```

Reply:
I've fix it.

Reply:
public enum Color